.build/tar/generate_tar.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-CONFIG_FILE=${CONFIG_FILE:-config-example.yaml}
+CONFIG_FILE=${CONFIG_FILE:-config-mysql_apache_jvm.yaml}
```

Comment:
Should we change the default config file to this?

Instead users can select which config to use when running the make command:
```bash
make CONFIG_FILE=config-mysql_apache_jvm.yaml build-tarball
```

Reply:
I thought with the current statement, the user can still specify which config to use. What we have here is just the default one in case the user doesn't specify a config file while running the make command?

Reply:
Just as Jingbo said, the user can still pass which config file they want, we're just here changing the default!

Reply:
Sorry my comment wasn't very clear. I was just asking why are we changing the default? Wouldn't the other one make more sense as the default? (relevant to more users)

Reply:
Yes, we'll revert to the other config as default.